### ISSUE TITLE
Support Twilio 4.x library (deprecation notices)

### ISSUE THREAD

**Issue by saurori** [twilio-labs/serverless-toolkit#473]
Hi, I recently upgraded my Twilio Functions dependency for `twilio` to `4.10.0` from `3.x` version and noticed a deprecation warning in all of my Function error logs:

```
services is deprecated. Use v1.services instead.
```

I believe this is from calling `Runtime.getSync()`. The warning is [here](https://github.com/twilio/twilio-node/blob/51599f28422a507f568046df3994675768b131fa/src/rest/Sync.ts#L9) and I believe it's called from [here](https://github.com/twilio-labs/serverless-toolkit/blob/0dde2a5a74035700e4ef6cf4b1c1189c78e2ff59/packages/twilio-run/src/runtime/internal/runtime.ts#L75).

Is there a plan to support `4.x` in the near future?

**Comment by AlaricWhitney** [comment]
I agree with this.  This library is being flagged by our security teams as associated with high severity CVE's, so we can't use it in it's current form.  the issue is that it's importing twilio v3, and needs to be importing twilio v4

**Comment by saurori** [comment]
@makserik Thanks for updating the library. However, the deprecation message still exists using version `4.23.0`:
```
services is deprecated. Use v1.services instead.
```

Please see the original description in this thread for where the deprecation warning is coming from.

**Comment by makserik** [comment]
@saurori - I believe this should have fixed the issue? https://github.com/twilio-labs/serverless-toolkit/pull/487/files

**Comment by saurori** [comment]
@makserik that seems to be working fine, using `twilio` `5.1.0` and `@twilio/runtime-handler` `2.0.1`

**Comment by saurori** [comment]
@makserik I guess I did not have enough log data, but the issue seems to still be present in `@twilio/runtime-handler` `2.0.1`. It looks like it was fixed in [2.0.2](https://github.com/twilio-labs/serverless-toolkit/releases/tag/%40twilio%2Fruntime-handler%402.0.2) but I cannot update. Setting the dependency to `2.0.2` gives the error:
<pre>Some dependencies could not be validated. No matching version found for @twilio/runtime-handler@2.0.2. You can find the valid versions at https://www.twilio.com/docs/serverless/functions-assets/handler</pre>

The [url in the error message](https://www.twilio.com/docs/serverless/functions-assets/handler) indicates `2.0.2` is not available. Is this a mistake?
